11-multi-track/bb_views_part1/readme.md

```diff
@@ -16,45 +16,52 @@
 * [Backbone Tutorials](https://backbonetutorials.com)
 * [Backbone.js for absolute beginners](http://adrianmejia.com/blog/2012/09/11/backbone-dot-js-for-absolute-beginners-getting-started/)
 
-## Intro - What are Backbone Views? (10 minutes)
+## Getting Started
 
-In Backbone, views are the objects responsible for:
+Today we will continue working on the Grumblr application we started building yesterday.  
 
-- Rendering content data into HTML, and appending it to the page
-- Listening to their models (for changes, destroys, etc), and responding accordingly
-- Listening for events on the page (usually within their `el`), and responding accordingly
+* [Starter Code](https://github.com/ga-dc/grumblr_backbone/tree/views_part_1_starter). Located on the `views_part_1_starter` branch.
+* [Solution Code](https://github.com/ga-dc/grumblr_backbone/tree/views_part_1_solution). Located on the `views_part_1_solution` branch.
 
```

Comment:
Starter code `index.html` references comment files that do not exist. Either create or delete references.